workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -14,7 +14,7 @@ public class AndroidClientWorkerConnector : MobileWorkerConnector, IMobileConnec
         public string IpAddress { get; set; }
         public ConnectionScreenController ConnectionScreenController { get; set; }
 
-        [SerializeField] private GameObject level;
+        public GameObject Level;
```

Comment:
This was changed when people said our standard is `[SerializeField] private` instead of `public`. What's the reasoning behind reverting this change?

Reply:
I hope 8 comments were enough to make my point clear